arm9/Makefile

```diff
@@ -208,24 +217,27 @@ $(BUILD_DIR)/%.o: dep :=
 endif
 
 # Cpp asm processor is not supported at this time.
-$(GLOBAL_ASM_O_FILES): BUILD_C := $(ASM_PROCESSOR) "$(CC) $(CFLAGS)" "$(AS) $(ASFLAGS)"
-BUILD_C ?= $(CC) -c $(CFLAGS) -o
+$(GLOBAL_ASM_O_FILES): BUILD_C := $(ASM_PROCESSOR) "$(CC) $(MWCFLAGS)" "$(AS) $(MWASFLAGS)"
+BUILD_C ?= $(CC) -c $(MWCFLAGS) -o
 
 $(C_OBJS): $(BUILD_DIR)/%.o: %.c $$(dep)
 	$(BUILD_C) $@ $<
 
 $(CXX_OBJS): $(BUILD_DIR)/%.o: %.cpp $$(dep)
-	$(CXX) -c $(CXXFLAGS) -o $@ $<
+	$(CXX) -c $(MWCXXFLAGS) -o $@ $<
 
 $(S_OBJS) $(LIBS_OBJS): $(BUILD_DIR)/%.o: %.s $$(dep)
-	$(AS) $(ASFLAGS) -o $@ $<
+	$(AS) $(MWASFLAGS) -o $@ $<
+
+$(BUILD_DIR)/$(LD_TEMPLATE): $(BUILD_DIR)/%: %
+	(echo "KEEP_SECTION\n{\n\t.exceptix\n}\n"; cat $<) > $@
```

Comment:
Can you elaborate on the issues?

Reply:
My echo, in this case, doesn't seem to interpret `\n` characters as newlines, or `\t` characters as tabs. It would just put `KEEP_SECTION\n{\n\t.exceptix\n}\n` at the top of the LCF.